### ISSUE TITLE
next task returns the latest task, not the oldest

### ISSUE THREAD

**Issue by benjyiw** [TnLCommunity/corndogs#58]
IMO the default intended behavior should be that corndogs returns the oldest task by the created time, not the newest task by the updated time. Could potentially make that configurable between created/updated, so that the user of the app can choose. However, the current ordering certainly seems like unintended behavior.

https://github.com/TnLCommunity/corndogs/blob/f87eea856137daf6e1359c8581a8218a8b03d447/server/store/postgresstore/postgresstore.go#L139

**Comment by JacobPuff** [comment]
Yeah, you're right, that's unintended. I'll put up a PR later tonight to fix it.

**Comment by JacobPuff** [comment]
### For starters, this project will not receive more updates, since it has been moved to Catalyst Community over here: https://github.com/catalystcommunity/corndogs

Anyways, apparently this isn't an issue, but it wasn't super clear and went against my initial instinct. [Postgres has their ORDER BY docs here](https://www.postgresql.org/docs/current/queries-order.html) which I recently looked at and misunderstood. Specifically it says:
> When more than one expression is specified, the later values are used to sort rows that are equal according to the earlier values.

So we're all good!

I confirmed this too before re-reading the docs, when I still misunderstood, by submitting these tasks using the corndogs CLI where `-r` specifies priority:
```
./corndogs submit-task -c start -l yo -r 2
./corndogs submit-task -c start -l foo -r 1
./corndogs submit-task -c start -l bar -r 2
./corndogs submit-task -c start -l YEET -r 1
./corndogs submit-task -c start -l negative -r -1
```

I then manually ran the query in pgAdmin which shows the correct priority order:

![Image](https://github.com/user-attachments/assets/883d9d66-94a4-4ea7-b136-7fe3ebdeec93)

**Comment by JacobPuff** [comment]
Actually, with discussion elsewhere, I was wrong. It's the *updated* time that is the issue. Which the screenshot shows is borked. It *should* be the oldest first, and it is not. It's giving the latest one as you mentioned.

I'll be fixing it in the new repo, but I'll link to that here when I'm done.

**Comment by JacobPuff** [comment]
PR for the fix *in the new repo* is here: https://github.com/catalystcommunity/corndogs/pull/28

**Comment by JacobPuff** [comment]
PR in the _new repo_ has now been merged.
